sim/plugins/MaavCameraPlugin.cpp

```diff
@@ -0,0 +1,167 @@
+#include <gazebo/common/common.hh>
+#include <gazebo/gazebo.hh>
+#include <gazebo/physics/physics.hh>
+#include <gazebo/plugins/DepthCameraPlugin.hh>
+#include <gazebo/rendering/DepthCamera.hh>
+#include <gazebo/rendering/rendering.hh>
+#include <gazebo/sensors/sensors.hh>
+#include <ignition/math.hh>
+
+#include <functional>
+#include <string>
+#include <vector>
+#include <Eigen/Core>
+
+#include <common/messages/MsgChannels.hpp>
+#include <common/messages/point_cloud_t.hpp>
+#include <common/messages/rgbd_image_t.hpp>
+#include <zcm/zcm-cpp.hpp>
```

Comment:
We no longer use ZCM. This file should be removed and replaced with the original ROS RealSense plugin we ripped this off from.